src/functions-and-arrays.js

```diff
@@ -1,18 +1,109 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(firstNumber, secondNumber)  {
```

Comment:
This is ok. You could also solve this using the ternary operator:
`function maxOfTwoNumbers(firstNumber, secondNumber)  {
  return firstNumber > secondNumber ? firstNumber : secondNumber
}
console.log(maxOfTwoNumbers(30, 30))`